lib/merge_hris_client/models/account_details_and_actions_status_enum.rb

```diff
@@ -30,6 +30,8 @@ def self.build_from_hash(value)
     # @param [String] The enum value in the form of the string
     # @return [String] The enum value
     def build_from_hash(value)
+      constantValues = AccountDetailsAndActionsStatusEnum.constants.select { |c| AccountDetailsAndActionsStatusEnum::const_get(c) == value }
+      raise "Invalid ENUM value #{value} for class #AccountDetailsAndActionsStatusEnum" if constantValues.empty?
```

Comment:
^ i believe these will break if the values dont match our enums (or are null)

Reply:
I removed these in the latest template tweaks